packages/desktop/src/renderer/components/widgets/update/UpdateModalComponent.test.tsx

```diff
@@ -116,7 +118,7 @@ describe('UpdateModal', () => {
                     <p
                       class="MuiTypography-root MuiTypography-body2 css-16d47hw-MuiTypography-root"
                     >
-                      Update is available for Quiet.
+                      UA new version of Quiet is ready. It will be installed the next time you restart the app, or you can update now.
```

Comment:
```suggestion
                      A new version of Quiet is ready. It will be installed the next time you restart the app, or you can update now.
```